compiler/generator.ml

```diff
@@ -41,6 +43,10 @@ let txt_of_binop = function
   | Greater -> ">"
   | Geq -> ">="
 
+(* Conditionals *)
+let txt_of_cond indent i t e = sprintf "if %s:\n%s\n%selse:\n%s\n" 
```

Comment:
since each if/else will be wrapped in one function we actually don't need the else